flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/CodeGenerator.scala

```diff
@@ -582,13 +562,14 @@ class CodeGenerator(
          |
          """.stripMargin
 
-    funcCode += genSetAggregationResults(accTypes, aggs, aggMapping) + "\n"
+    funcCode += genSetAggregationResults(accTypes, aggs, aggMapping, partialResults) + "\n"
```

Comment:
If we really want to indicate the needed for each method, we should include all parameters, for instance, genSetAggregationResults needs groupingKeys. So to make it simple, I think we do not need to pass those to genXXX functions.